inventory-generation/notifications/templates/list-of-users.yaml.j2

```diff
@@ -26,6 +26,17 @@ internal:
 {{ users.append(technical_lead_email) }}
 {%- endif %}
 
+admins:
+  list_of_users:
+{% for user in engagement_users | default([]) %}
+{% filter indent(width=6) %}
+{% if 'lodestar-admins' in user.role %}
```

Comment:
`lodestar-admins` isn't a good value to key off here. Now that we have the runtime.json in the engagement repos, we should look to use that to flag a user/groups of users for addition here. 

Example usage of `runtime.json`:
https://github.com/rht-labs/labs-sre-automation/blob/master/inventory-generation/identity-management/main.yml#L19